dist/s_all

```diff
@@ -91,7 +91,7 @@ run "python test_tag.py"
 # The s_mentions script requires bash.
 run "./s_mentions" "--warning-only"
 if command -v evergreen > /dev/null; then
-	run "evergreen validate ../test/evergreen.yml"
+	run "evergreen validate -p wiredtiger ../test/evergreen.yml"
```

Comment:
Though we have Evergreen projects other than "wiredtiger" for stable release branches (e.g. mongodb-6.0), a simple fix is to validate against the "wiredtiger" develop Evergreen project as the project-level configurations are quite similar among all wiredtiger Evergreen projects.